source/menu/events/add.ts

```diff
@@ -93,36 +90,101 @@ menu.choose('a', {
 	columns: RESULT_COLUMNS,
 	async choices(ctx) {
 		try {
-			const all = await findEvents(ctx);
-			return Object.fromEntries(all.map(event => [event.replaceAll('/', ';'), event]));
+			const filteredEvents = findEvents(ctx);
+			const alreadySelected = Object.keys(ctx.userconfig.mine.events);
+
+			ctx.session.eventDirectorySubDirectoryItems = Object.keys(filteredEvents.subDirectories);
+			const subDirectoryItems = Object.entries(filteredEvents.subDirectories)
+				.map(([name, directory], i) =>
+					directory.subDirectories !== undefined || directory.events !== undefined
```

Comment:
Nicht sicher, ob ich die Logik hier richtig verstehe. Vielleicht wäre sowas sinnvoller? Wenn eins von beiden nicht leer ist, dann Ordner?

```suggestion
					Object.keys(directory.subDirectories ?? {}).length > 0 || Object.keys(directory.events ?? {}).length > 0
```

Vermutlich wärs aktuell die gleiche Logik, weil die directory.json vermutlich undefined enthält, wenns keine Keys hat? Wenn ich nichts falsch verstanden hab fänd ich den Ansatz über die Anzahl der keys zu gehen aber leichter lesbar im Sinne von, was ist das Ziel? Übersehe ich was? Oder bist du anderer Meinung?